openmp/libomptarget/unittests/Plugins/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+set(LIBOMPTARGET_BUILD_AMDGPU_PLUGIN TRUE CACHE BOOL
+  "Whether to build AMDGPU plugin")
+set(LIBOMPTARGET_BUILD_CUDA_PLUGIN TRUE CACHE BOOL
+  "Whether to build CUDA plugin")
+
+set(PLUGINS_TEST_COMMON omptarget OMPT omptarget.devicertl)
+set(PLUGINS_TEST_SOURCES NextgenPluginsTest.cpp)
+set(PLUGINS_TEST_INCLUDE ${LIBOMPTARGET_INCLUDE_DIR})
+
+if (LIBOMPTARGET_BUILD_AMDGPU_PLUGIN)
```

```suggestion
if(TARGET omptarget.rtl.amdgpu)
```
We should be able to just check the existence of the target so long as we add the unit tests after the main source.